forem/forem
# Fix "Infinity" dashboard analytics by using 2 digits for hardcoded month and day for JS Date

## What type of PR is this? (check all applicable)


## Description

Single-digit months and days are not compliant with the ECMAScript
standard for date strings.

See: https://262.ecma-international.org/11.0/#sec-date-time-string-format

Fixes #13217 

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

1. Open Safari
2. Visit https://dev.to/dashboard/analytics
3. Click "Infinity"
4. You should see your metrics since April 1, 2019

### UI accessibility concerns?

None, just a date format fix

## Added tests?

- [x] No, and this is why: If we were accepting arbitrary dates here, I think it would make sense to write a test, but this changes a hardcoded value to match the ECMAScript spec

## [Forem core team only] How will this change be communicated?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
